crates/noirc_frontend/src/hir/resolution/resolver.rs

```diff
@@ -884,6 +895,41 @@ impl<'a> Resolver<'a> {
                 }
                 Literal::Integer(integer) => HirLiteral::Integer(integer),
                 Literal::Str(str) => HirLiteral::Str(str),
+                Literal::FmtStr(str) => {
```

Comment:
Also since this case is large, lets break it out into a function.

Reply:
done